### PR TITLE
Use MS Build of OpenJDK

### PR DESCRIPTION
**Pull request by brunoborges**
Hey @DarqueWarrior @artursouza, here's an update to use the MS Build of OpenJDK.

### CODE REVIEW

File: twitter-sentiment-processor/demos/javademo/processor/Dockerfile [hunk 1]
```diff
@@ -1,21 +1,34 @@
...
-VOLUME /tmp
 WORKDIR /build
+COPY pom.xml ./
+ADD src /build/src
+RUN mvn package
 
-COPY pom.xml .
-RUN mvn dependency:go-offline
+# Base Image
+FROM ubuntu:20.04
```
Comment (artursouza) [inline twitter-sentiment-processor/demos/javademo/processor/Dockerfile:10]: Is there a distroless base?

File: twitter-sentiment-processor/demos/javademo/processor/Dockerfile [hunk 2]
```diff
@@ -1,21 +1,34 @@
...
 
-ADD src/ /build/src/
-RUN mvn package
+# Download and Extract the Microsoft Build of OpenJDK from the Microsoft OpenJDK website
+RUN apt-get update && \
```
Comment (artursouza) [inline twitter-sentiment-processor/demos/javademo/processor/Dockerfile:13]: Does MS plan to offer a distroless base image with the JDK pre installed, similar to the one we use from g?

File: twitter-sentiment-processor/demos/javademo/provider/Dockerfile [hunk 3]
```diff
@@ -1,21 +1,34 @@
...
-VOLUME /tmp
 WORKDIR /build
+COPY pom.xml ./
+ADD src /build/src
+RUN mvn package
 
-COPY pom.xml .
-RUN mvn dependency:go-offline
+# Base Image
+FROM ubuntu:20.04
```
Comment (artursouza) [inline twitter-sentiment-processor/demos/javademo/provider/Dockerfile:10]: Same.

File: twitter-sentiment-processor/demos/javademo/viewer/Dockerfile [hunk 4]
```diff
@@ -1,21 +1,34 @@
...
-VOLUME /tmp
 WORKDIR /build
+COPY pom.xml ./
+ADD src /build/src
+RUN mvn package
 
-COPY pom.xml .
-RUN mvn dependency:go-offline
+# Base Image
+FROM ubuntu:20.04
```
Comment (artursouza) [inline twitter-sentiment-processor/demos/javademo/viewer/Dockerfile:10]: Same.

**Review by artursouza — CHANGES_REQUESTED** [review]
Thanks for the contribution. I just have a question about the base image.

File: twitter-sentiment-processor/demos/javademo/processor/Dockerfile (hunk 1 above)
Comment (brunoborges) [reply twitter-sentiment-processor/demos/javademo/processor/Dockerfile:10]: Not at the moment.

File: twitter-sentiment-processor/demos/javademo/processor/Dockerfile (hunk 2 above)
Comment (brunoborges) [reply twitter-sentiment-processor/demos/javademo/processor/Dockerfile:13]: We are in the process of defining our container image deliverables. In the short term, we will not be publishing a distroless image.

**Review by artursouza — APPROVED** [review]

**Pull request merged**